common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1408,3 +1417,32 @@ def clean_id(self, padding_char='='):
         return "course_{}".format(
             b32encode(unicode(self.location.course_key)).replace('=', padding_char)
         )
+
+    @property
+    def teams_enabled(self):
```

Comment:
I was thinking of this as the per-course setting, but I think that's a good idea, especially if we have teams configurations that get imported into a system where teams aren't enabled, etc.